integrations/satismeter/lib/index.js

```diff
@@ -25,10 +25,19 @@ var SatisMeter = (module.exports = integration('SatisMeter')
 
 SatisMeter.prototype.initialize = function() {
   var self = this;
+  var options = this.options;
   this.load(function() {
-    when(function() {
-      return self.loaded();
-    }, self.ready);
+    when(
+      function() {
+        return self.loaded();
+      },
+      function() {
+        window.satismeter('load', {
+          writeKey: options.apiKey || options.token
```

Comment:
@E-Mud Maybe we could add `source` here. We had a separate PR for this, but it's not in the current repo.

Reply:
Good catch. I completely forgot about `source`. Done -> 731df99